templates/applicantForm.phtml

```diff
@@ -164,6 +183,12 @@
 
                 <div id="onboarding" class="collapse">
 
+                    <label for="fee">Course Fee</label>
+                    <div class="input-group">
+                        <span class="input-group-addon">&pound;</span>
+                        <input id="fee" name="fee" class="submitApplicant form-control" type="number" placeholder="0" min="0" max="10000" step="1" value="<?php echo $applicant->getFee();?>">
+                    </div>
+
```

Comment:
I seem to have an issue here where I'm getting the "Total payment is more than course price" error when doing this:![Screenshot 2021-05-26 at 15 14 50](https://user-images.githubusercontent.com/19924378/119675703-2f3cf400-be35-11eb-8cf3-dac1640d75dc.png)